src/main/java/io/jenkins/infra/repository_permissions_updater/hosting/MavenVerifier.java

```diff
@@ -133,8 +133,8 @@ private void checkGroupId(Model model, HashSet<VerificationMessage> hostingIssue
         try {
             String groupId = model.getGroupId();
             if(StringUtils.isNotBlank(groupId)) {
-                if(groupId.equals("org.jenkins-ci.plugins")) {
-                    hostingIssues.add(new VerificationMessage(VerificationMessage.Severity.REQUIRED, SHOULD_BE_IO_JENKINS_PLUGINS));
+                if (groupId.equals("org.jenkins-ci.plugins") || !groupId.equals("io.jenkins.plugins")) {
```

Comment:
this would be simpler:
```suggestion
                if (!groupId.equals("io.jenkins.plugins")) {
```

Reply:
true dat